elife/config/etc-php-7.0-fpm-php.ini

```diff
@@ -1757,7 +1757,7 @@ ldap.max_links = -1
 
 ; When disabled, you must reset the OPcache manually or restart the
 ; webserver for changes to the filesystem to take effect.
-;opcache.validate_timestamps=1
+opcache.validate_timestamps={% if pillar.elife.dev %}1{% else %}0{% endif %}
```

Comment:
This means that any PHP application will have to restart PHP-FPM for changes to be recognised.

Reply:
We shouldn't modify files manually very often.
During deploy/Salt highstates, https://github.com/elifesciences/builder-base-formula/blob/master/elife/nginx-php7.sls does not restart the service by default. States that update the files need a `watch_in` to trigger the restart https://github.com/elifesciences/journal-formula/blob/master/salt/journal/init.sls#L14-L15 before this can change can be made.